src/index.html

```diff
@@ -17,11 +17,472 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.css"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header_bar">
+      <a
+        class="logo"
+        href="#logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo MOYO"
+        />
+      </a>
+
+      <nav class="navigation_bar">
+        <ul class="navigation_bar__list">
+          <li class="navigation_bar__link">
+            <a
+              href="#apple"
+              class="link is-active"
```

Comment:
[The name of an element must contain the name of its block](https://mate-academy.github.io/fe-program/css/typical-bem-mistakes-en#using-an-element-without-its-blocks-prefix)